common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -672,8 +701,27 @@
         /**
          * The newest BrokerAPI-To-Broker protocol version.
          */
-        public static final String BROKER_API_TO_BROKER_PROTOCOL_VERSION_CODE = "1.0";
-
+        public static final String BROKER_API_TO_BROKER_PROTOCOL_VERSION_CODE;
+
+        static {
+            float protocolVersion = 1.0f;
+            String stringVersion = "1.0";
```

Comment:
by just tracking the string in parallel, I don't have to try to format the numeric which is filled with edge cases.